src/shared/models/asset.model.ts

```diff
@@ -0,0 +1,51 @@
+import type BigNumber from 'bignumber.js';
+
+import { WalletChainTypes } from './blockchain.model';
+
+export enum WalletAssets {
+  ft = 'ft',
+  nft = 'nft',
+}
```

Comment:
I would advocate we just use a string here, this way, we can import all of this code with `import types`, rather than actual code (better with the shared script context set up). There's no real need for us to use an `enum`.

```suggestion
export type WalletAssetType = 'ft' | nft';
```

Reply:
But I do use it as an enum in places? 🤔  `WalletAssets.ft`? Maybe it became unneeded, but I'll check.

Reply:
I see the benefit now here, will change. I had thought this was the preferred way to do it. I did it this way in `WalletBlockchains` too so I'll fix.